cypress/integration/all-flaky-tests/subcase.spec.js

```diff
@@ -107,6 +107,7 @@ context('Subcase tests', () => {
     cy.visit('/dossiers/5F02E3F87DE3FC0008000002/deeldossiers');
     cy.addSubcase(type, shortSubcaseTitle, subcaseTitleLong, subcaseType, subcaseName);
     cy.openSubcase(0);
+    cy.wait(2000);
```

Comment:
also stated in the commit message but wanted to reiterate here:

Not the ideal solution, but it seems to fix an error that's caused by
Cypress immediately deleting the subcase while some requests are still
in flight.

Error at hand:
```
1) Subcase tests
    should add a subcase and then delete it:
   Error: The following error originated from your application code, not from Cypress.

> Attempted to handle event `pushedData` on <subcase:629675108AADBDDBE93A3D46> while in state root.deleted.inFlight.
```
I am guessing that the timing of the requests are off, and that perhaps
the way computed properties work make it so deleting the subcase record
while the requests are still in flight causes an error here.

Reply:
if this is caused by deleting subcase, it would make more sense to await in the test where we acutally delete the subcase. Will look at this when reviewing